projects/packages/jetpack-mu-wpcom/src/features/custom-css/csstidy/class.csstidy-optimise.php

```diff
@@ -632,11 +632,11 @@ public static function explode_ws( $sep, $string ) {
 	 * @see dissolve_4value_shorthands()
 	 */
 	public static function merge_4value_shorthands( $array ) {
-		$return     = $array;
+		$return     = is_array( $array ) ? $array : array();
 		$shorthands = & $GLOBALS['csstidy']['shorthands'];
 
 		foreach ( $shorthands as $key => $value ) {
```

Comment:
An example of value for `$key => $value` is  `'padding' => [ 'padding-top', 'padding-right', 'padding-bottom', 'padding-left' ]`